### ISSUE TITLE
List members of the steering group and maintainers on the website

### ISSUE THREAD

**Issue by Remi-Gau** [bids-standard/bids-website#167]
@gllmflndn mentioned that this information is "a bit" hidden and could maybe benefit from being more visible on the website itself.

In the governance page?

**Comment by sappelhoff** [comment]
agreed, this is a bit hidden (unintentionally).

I think it's currently only here: https://github.com/bids-standard/bids-specification/blob/master/DECISION-MAKING.md#steering-group

And on a newitem here: https://bids.neuroimaging.io/news.html

The governance page would be a natural place in my opinion as well.

**Comment by gllmflndn** [comment]
Thanks for opening an issue about this, @Remi-Gau!

The governance page was the first place I looked at and it seemed out of date:
> The project is currently managed and maintained by Franklin Feingold, Stefan Appelhoff, and the Poldrack Lab at Stanford.

and it was only after some further search that I found more information in `DECISION-MAKING.md`. It would be great to have all of the members of the steering and maintainers groups listed more prominently on the website.

For the background to this, it started by me wondering who was the `our` in [@franklin-feingold's comment](https://github.com/bids-standard/bids-website/pull/157#issue-500867712). It might also be useful to mention somewhere who holds the keys of the project (who owns the neuroimaging.io domain name, the Google Analytics account and discussion list, etc).

**Comment by sappelhoff** [comment]
> It might also be useful to mention somewhere who holds the keys of the project (who owns the neuroimaging.io domain name, the Google Analytics account and discussion list, etc).

Agreed! We should do a better job at transparency here.

**Comment by gllmflndn** [comment]
> Agreed! We should do a better job at transparency here.

Oh, I think you are already doing a very good job at transparency - I am perhaps mentioning the last remaining little details.

I might be hijacking this thread instead of opening another issue but, as I mentioned the website already, wouldn't it make sense to host BIDS on a new something bids something standard something `.org` instead of [neuroimaging](http://neuroimaging.io/)[.io](https://en.wikipedia.org/wiki/.io#Controversy)?
